fili-core/src/main/java/com/yahoo/bard/webservice/data/HttpResponseChannel.java

```diff
@@ -58,16 +35,15 @@ public HttpResponseChannel(
      * @param apiRequest  Api request object with all the associated info with it
      * @param httpResponseMaker  Helper class instance to prepare the response object
      *
-     * @deprecated  The ResponseFormatType and UriInfo should be passed explicitly, rather than implicitly via the
```

Comment:
The primary purpose of making the `HttpResponseChannel` injectable is so that customers can inject their own `HttpResponseChannel` so that they can customize how they send back the response. 

There are situations (like the one that motivated this change) where customers will want to customize the response format based on something in the request. However, we can't very well predict what components of the request customers will use to determine how to send the response.